base/setup/usetup/consup.c

```diff
@@ -692,11 +692,11 @@ CONSOLE_SetStyledText(
 
     if (Flags & TEXT_ALIGN_CENTER)
     {
-        coPos.X = (xScreen - strlen(Text)) / 2;
+        coPos.X = (SHORT)((xScreen - strlen(Text)) / 2);
```

Comment:
Or better:
`coPos.X = (xScreen - (SHORT)strlen(Text)) / 2;`
(and similar for the other one)